docs/native/api/goal-tracking.md

```diff
@@ -1,7 +1,7 @@
 
 # Goal Tracking
 
-Conversions goals help you measure the performance of aspects of your site. With Altis Native Analytics you can use the [client-side API](./client-side-api.md) to track any events you wish, however, some goal tracking is integrated into the [Experiments](./experiments.md) and [Experience Blocks](./experience-blocks.md) features.
+Conversions goals help you measure the performance of aspects of your site. With Altis Native Analytics you can use the [client-side API](./client-side-api.md) to track any events you wish, however, some goal tracking is integrated into [Altis Optimization Framework](../../optimization-framework/) features like [A/B Testing](../../optimization-framework/ab-testing.md) and [Experience Blocks](../../optimization-framework/experience-blocks.md).
```

Comment:
I think this goal tracking document should live under the Optimization Framework section to be honest as it's really tied to that. The native analytics portion itself doesn't really do anything with it by default.

Reply:
I had updated the link in the `ab-testing.md` but didn't move the file, good catch!